flask_monitoringdashboard/static/elements/pagination.html

```diff
@@ -2,6 +2,42 @@
     <div class="col-sm-6">
         Displaying <strong>{{ pagination.getLeft() + 1 }}
         - {{ pagination.getRight() }}</strong> {{ pagination.name }}: <strong>{{ pagination.total | number }}</strong> in total
+        <div class="pagination" style="float: left;">
+            <h6>Sort By:</h6>&nbsp;&nbsp;
+            <div>
+                <a href="#" ng-click="pagination.updatesort('time_requested', 'Latest')">Latest</a>
+            </div>
+            <div>
+                <h6>|</h6>
+            </div>
+            <div>
+                <a href="#" ng-click="pagination.updatesort('duration', 'Duration')">Duration</a>
+            </div>
+            <div>
+                &nbsp;&nbsp;(Sorted by: <strong> {{ pagination.showValue }}</strong>)
+            </div>
+        </div>
+        <div class="pagination" style="float: left;">
+            <h6>Response Code:</h6>&nbsp;&nbsp;
+            <div>
+                <a href="#" ng-click="pagination.updateStatusCode('all')">All</a>
+            </div>
+            <div>
+                <h6>|</h6>
+            </div>
+            <div>
+                <a href="#" ng-click="pagination.updateStatusCode('success')">Success</a>
+            </div>
+            <div>
+                <h6>|</h6>
+            </div>
+            <div>
+                <a href="#" ng-click="pagination.updateStatusCode('failed')">Failed</a>
```

Comment:
either Success | Fail or Succeed*ed* | Fail*ed*

Reply:
The way Buttons are there is at least one of them will be selected so that's why All is there to filter all irrespective of the response code. So I would suggest not changing this.

I have made other changes already, please do review them.

Reply:
Perfectly find that All is there. 
I was referring to the *form* of Sucess / Fail. 
It should match: if the first is Success then the second should be Fail. 
Or Succeeded / Failed. But if you mix them it looks bad.